Options/Applicative/Types.hs

```diff
@@ -384,7 +384,9 @@ data OptHelpInfo = OptHelpInfo
 data OptTree a
   = Leaf a
   | MultNode [OptTree a]
-  | AltNode [OptTree a]
+  | AltNode Bool [OptTree a]
+  -- ^ The 'Bool' argument indicates whether this node should be displayed with
```

Comment:
You would probably be better off making a type for this instead of using Bool

Reply:
Done